classes/core-image-pelux.bbclass

```diff
@@ -3,8 +3,6 @@
 #   SPDX-License-Identifier: MIT
 #
 
-DESCRIPTION = "Image for creating a Pelux image"
-
```

Comment:
Unrelated change, please remove (from this patch).

Reply:
This one is not unrelated. The diff is misleading because git detects the addition of this new file as a move from the core-image-pelux.bb, since the contents are indeed mostly the same.

But in fact, I didn't remove these lines: simply didn't copy them into the new file, as they should not be added there.

Reply:
ah, the move wasn't obvious to me.